app/code/core/Mage/Sales/Model/Order.php

```diff
@@ -2129,6 +2129,11 @@ public function getInvoiceCollection()
         return $this->_invoices;
     }
 
+    public function getInvoicesCollection()
```

Comment:
I'd suggest adding a doc block here otherwise I'm sure @sreichel will do it later :D

Reply:
```suggestion
    /**
     * Retrieve order invoices collection
     *
     * @return Mage_Sales_Model_Resource_Order_Invoice_Collection
     */
    public function getInvoicesCollection()
```